### ISSUE TITLE
Add Meteor Community Packages to maintainers

### ISSUE THREAD

**Issue by StorytellerCZ** [Meteor-Community-Packages/meteor-link-accounts#84]
@yubozhao Would you be so kind and add Meteor Community Packages group to maintainers on Atmosphere as described, so that we don't have to bother you about every update?
https://github.com/Meteor-Community-Packages/organization#process-of-adding-a-package

So that should be:
`meteor admin maintainers bozhao:link-accounts --add communitypackages`

**Comment by yubozhao** [comment]
Done
```
Adding a maintainer to bozhao:link-accounts...

The maintainers for bozhao:link-accounts are:
bozhao
communitypackages```

**Comment by yubozhao** [comment]
@StorytellerCZ I also published the latest version `v2.1.1`

**Comment by StorytellerCZ** [comment]
Awesome! Can you also send me where people can send you donations, right now I implemented mine from the template package (see sponsor button at the top), but those should be yours.
